src/Microsoft.AspNetCore.Mvc.ViewFeatures/DependencyInjection/MvcViewFeaturesMvcCoreBuilderExtensions.cs

```diff
@@ -105,6 +105,11 @@ internal static void AddViewServices(IServiceCollection services)
             services.TryAddSingleton<IHtmlGenerator, DefaultHtmlGenerator>();
 
             //
+            // Expression Helper
+            //
```

Comment:
This doesn't need it's own 'block' here it's not a top-level feature. Group it with `IHtmlGenerator` and the other misc infrastructure.